props/saber_fett263_G_buttons.h

```diff
@@ -0,0 +1,445 @@
+// Fett263 Buttons with Gesture Controls and Battle Mode with Smart Lockup
+//
+// Includes Gesture Controls, "Battle Mode" with "Smart Lockup", "Multi-Blast" Mode (to enable Swing Blast control) and SA22C volume menu
+// Optional Defines
+//
+// FETT263_STAB_ON 
+// To enable Stab On
+//
+// FETT263_TWIST_ON 
+// To enable Twist On
+//
+// FETT263_MULTI_PHASE
+// This will enable "live" preset change to create a "Multi-Phase" saber effect with preset changes on the fly while blade is ignited.
+//
+// LOCKUP_THRESHOLD_G 2.0
+// This is the "delay" threshold to determine Clash vs Lockup
+//
+// 2 Button Controls (PWR and AUX):
+// "Battle Mode" - Swing while saber is OFF to ignite in Battle Mode
+// or hold AUX and Swing while blade is ON to toggle mode
+//
+// Ignite (ON) - click PWR while OFF or Swing Saber while OFF
+//
+// Muted Ignition (ON) - double click PWR while OFF
+// Retract (OFF) - click PWR while ON (not swinging or in Battle Mode)
+// Twist Off (OFF) if in Battle Mode
+// Play Music Track - hold and release PWR while OFF
+// Blast - click AUX while ON
+// Multi-Blast Mode - hold and release AUX while ON to enter mode, Swing to initiate Blasts, click Aux to exit mode
+// Clash - clash blade while ON
+// Stab - thrust forward and clash blade while ON
+// Lockup - hold AUX and clash while ON
+// Drag - hold AUX and clash pointing down while ON
+// Lightning Block - hold PWR and click AUX while ON
+// Melt - hold PWR (or AUX) and thrust forward and clash while ON
+// Force - hold and release PWR while ON
+// Color Change - hold AUX and click PWR while ON to enter CCWheel,
+// turn hilt to rotate through colors, click PWR to select/exit
+// if using COLOR_CHANGE_DIRECT each button press advances one Color at a time
+// Next Preset - click AUX while OFF
+// MULTI_PHASE Next Preset - hold AUX and TWIST while ON (use define to enable)
+// Previous Preset - hold AUX and click PWR while OFF
+// MULTI_PHASE Previous Preset - hold PWR and TWIST while ON (use define to enable)
+// Enter SA22C Volume Menu - hold and release AUX while OFF
+// Volume Up (10% increment, 100% max) - click PWR while in Volume Menu while OFF
+// Volume Down (10% increment) - click AUX while in Volume Menu while OFF
+// Exit Volume Menu - hold and release AUX while in Volume Menu while OFF
+
+#ifndef MOTION_TIMEOUT
+#define MOTION_TIMEOUT 60 * 15 * 1000
+#endif
+
+#ifndef LOCKUP_THRESHOLD_G
+#define LOCKUP_THRESHOLD_G 2.0
+#endif 
+
+#ifndef PROPS_SABER_FETT263_G_BUTTONS_H
```

Comment:
This should be first. (Either at the top of the file, or right after the comments.)

Reply:
fixed